### PR TITLE
[W5.11][W15-A1]Muhammad Nur Kamal - Enhancement 2

### PR DESCRIPTION
**Pull request by MuhdNurKamal**
Enhancement 2 out of 2 for W5.11
Added sort method to display contacts in lexicographic order

### CODE REVIEW

**Comment by jas5469** [comment]
Very clear and effective sorting method. Unlike my method it actually sorts the list in the address book. Input and output are added comprehensively as well.

**Comment by jas5469** [comment]
would you mind to share how did you sort the inner List as well, as i could not in my case. Thanks!

**Comment by MuhdNurKamal** [comment]
To change internal list, change the sort allPersons in AddressBook

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -84,4 +84,10 @@ public String toString() {
         return getAsTextShowAll();
     }
 
+    @Override
+    public int compareTo(Person other) {
+        if(other == null)
+            return 1;
```
Comment (devamanyu) [inline src/seedu/addressbook/data/person/Person.java:90]: Instead of "1", use a constant or boolean variable such as TRUE.

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (devamanyu) [inline src/seedu/addressbook/parser/Parser.java:14]: Avoid auto-formatting unrelated parts of the code. Doing so can result in unrelated changes being included in the commit. You can use Sourcetree to look through changes before committing, and discard any unrelated changes.

**Review by devamanyu — COMMENTED** [review]
Good efficient implementation. Please close this PR after reading the minor comments.
